src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -55,11 +55,10 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     let customInit = config.init || _.noop;
 
     // optional search source which this object configures
-    self.searchSource = config.searchSource && new SearchSource();
+    self.searchSource = config.searchSource ? new SearchSource() : null;
```

Comment:
I disagree with parts of both sides :)

I **do** like Stacy's style... I much rather have a one liner than introduce another if branch in the code (if/else branches account to noise)

I **don't** like the use of `null` and I much rather only using `undefined`. Yes, I know that some argue against it and say you shouldn't do it, but I'm +100 to also explicitly assigning it when appropriate. Why? because it's explicit. JS as a lang as this core characteristic that variable values are hidden assignments are hard to follow. The more explicit we have things, the easier it is to read & understand the code.